core/src/main/resources/AppConstants.properties

```diff
@@ -361,8 +361,9 @@ msg.log.hidden.default=true
 msg.log.keys=conversationId
 # Define xPath for key conversationId
 msg.log.xPath.conversationId=*[local-name()='Envelope']/*[local-name()='Header']/*[local-name()='MessageHeader']/*[local-name()='HeaderFields']/*[local-name()='ConversationId']
+msg.log.xPath.value=/test/value
```

Comment:
Deze moet hier niet maar in de DeploymentSpecifics van iaf-test

Reply:
PS @jkosternl Dat is een vergissing die ik in het oorspronkelijke PR gemaakt had en die toen niet opgemerkt was.